src/style.css

```
@@ -1 +1,38 @@
-/* styles go here */
+.label-group {
+  display: block;
+}
+
+.fieldset-1 {
+  margin-top: 10px;
+  margin-bottom: 0;
+}
+
+.fieldset-2 {
+  margin-top: 20px;
+  margin-bottom: 0;
+}
+
+.fieldset-3 {
+  margin-top: 20px;
+  margin-bottom: 0;
+}
+
+.fieldset-4 {
+  margin-top: 20px;
+  margin-bottom: 0;
+}
+
+button {
+  margin-top: 20px;
+  margin-bottom: 0;
+}
+
+input {
+  margin-top: 5px;
+  margin-bottom: 5px;
+}
+
+.additional {
+  margin-top: 10px;
+  margin-bottom: 0;
+}
```

add empty line at the end of file 
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file
```suggestion
}

```